src/containers/Validators/index.tsx

```diff
@@ -24,6 +24,9 @@ import NetworkContext from '../shared/NetworkContext'
 import { VALIDATOR_ROUTE } from '../App/routes'
 import { buildPath, useRouteParams } from '../shared/routing'
 import { VotingTab } from './VotingTab'
+import logger from '../../rippled/lib/logger'
+
+const log = logger({ name: 'ledgers' })
```

Comment:
I would update the name to be more specific to this page. Maybe just `validator`.